Gemfile.lock

```diff
@@ -37,22 +38,23 @@ GEM
       rspec-core (~> 3.9.0)
       rspec-expectations (~> 3.9.0)
       rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.0)
-      rspec-support (~> 3.9.0)
-    rspec-expectations (3.9.0)
+    rspec-core (3.9.2)
+      rspec-support (~> 3.9.3)
+    rspec-expectations (3.9.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-mocks (3.9.0)
+    rspec-mocks (3.9.1)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.9.0)
-    rspec-support (3.9.0)
-    sequel (5.27.0)
+    rspec-support (3.9.3)
```

Comment:
### [rspec-support](https://github.com/rspec/rspec-support)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 3.9.0 → 3.9.3

[_[change-log](https://github.com/rspec/rspec-support/blob/v3.9.3/Changelog.md), [source-code](https://github.com/rspec/rspec-support)_]





<details>
<summary>Commits</summary>

A change of **19** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-support/compare/v3.9.0...v3.9.3).

These are the first 10 commits:
- (97ea7a0) [Merge pull request #392 from rspec/update-travis-build-scrip…](https://github.com/rspec/rspec-support/commit/97ea7a0471f1a565e668ce7bb358c498ea21f1dd)
- (142fb3c) [Updates from rspec-dev that were missed](https://github.com/rspec/rspec-support/commit/142fb3ceb47ddfff3750b4f24b41e7eb2a383bdd)
- (6810564) [Version 3.9.1](https://github.com/rspec/rspec-support/commit/681056417db7d6b07f46b5bb6cb2fd16303fe281)
- (db9e3b7) [Add missing changeog link](https://github.com/rspec/rspec-support/commit/db9e3b7ef9086ee624b6f420e11ec70d4f8e2a83)
- (54cb8c2) [Merge pull request #394 from mvz/test-hashes](https://github.com/rspec/rspec-support/commit/54cb8c2e24c9eb6e8c675099c6a40d611e9d3a01)
- (70606e6) [Changelog for 3.9.2](https://github.com/rspec/rspec-support/commit/70606e6dffa31bc77151040134f2c8fa0c7aba91)
- (a5726e0) [Add missing full changelog links](https://github.com/rspec/rspec-support/commit/a5726e0ee600ee1d329f3d583c8dd6d56b437b37)
- (17a63b5) [Version 3.9.2](https://github.com/rspec/rspec-support/commit/17a63b50b23972ceeb7ff40b880a46b9b323c956)
- (3fd32d5) [Sync build with master](https://github.com/rspec/rspec-support/commit/3fd32d5a5e2e88cf8d9b87cacea0ea9d015e5e3b)
- (6c5d192) [Merge pull request #395 from bjfish/update-ripper-helper-and…](https://github.com/rspec/rspec-support/commit/6c5d192df884d700a38bb7af0f4c69b40de808ec)


</details>